src/screens/SearchResultsScreen.tsx

```diff
@@ -8,7 +11,7 @@ import { ShowCard, ShowCarousel } from '../components';
  * @param request | HTTP GET request from the SearchInput component
  * @returns {Promise<string>} | the users query
  */
-export async function loader({request}: {request: Request} ): Promise<string> {
+export async function loader({ request }: { request: Request }) {
```

Comment:
Spaces are fine but we should add in the return type, maybe just got lost in a rebase